src/main/java/Tasks/TaskList.java

```diff
@@ -0,0 +1,80 @@
+package Tasks;
+
+import Exceptions.DukeException;
+import Tasks.Task;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Returns the total number of lists.
+     * @return size of list as an integer
+     */
+    public int getListSize() {
+        return tasks.size();
+    }
+
+    public String getTaskString(int index) {
+        return tasks.get(index).toString();
+    }
+
+    /**
+     * Finds the task of which description contains the key string.
+     * @param key keyword inputted by user
+     * @return list of tasks that contains the keyword
+     */
+    public ArrayList<Task> findTasks (String key) {
+        ArrayList<Task> temp = new ArrayList<>();
+        for (Task t: tasks) {
+            if (t.getDescription().contains(key)) {
+                temp.add(t);
+            }
+        }
+        return temp;
+    }
+
+    /**
+     * Adds a new task to the list of tasks
+     * @param task task to be added
+     * @throws DukeException
+     */
+    public void addNewTask(Task task) throws DukeException {
+        tasks.add(task);
+    }
+
+    /**
+     * Removes a task from the list of tasks and returns it.
+     * @param index index of file to be deleted
+     * @return deleted task
```

Comment:
The Javadoc format doesn't seem correct, you can explain what is being returned by this method.